package.json

```diff
@@ -84,6 +84,7 @@
     "react-redux": "4.4.5",
     "rebass": "0.3.0",
     "redux": "3.5.2",
-    "redux-thunk": "2.1.0"
+    "redux-thunk": "2.1.0",
+    "reselect": "^3.0.1"
```

Comment:
currently working on some version freezing guidelines but I think we'll want to freeze the version to `3.0.x`